src/CycloneDX.Core/CycloneDX.Core.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.0;net5.0</TargetFramework>

```

Comment:
You are correct it should be TargetFrameworks. 

In relation to which framework we add, we have an interesting situation which I would like your input on based on some discussions I have had with other teams.

If I go with
- the current proposed TFM will ensure STJ V7 is used for net core 3 + standard, STJ V5 is used for net 5, STJ V6 for net 6 etc. Some people will consider that a breaking change
- dropping TFM to netcore 3, then same as above except net core 3 uses STJ v3 so also potential breaking change.
- update TFM to net 7 which will ensure all clients use STJ v7 or higher.
- update TFM to net 6 & decrease dependency to v6 so all clients are using atleast v6. But potential a breaking change but easy to solve. Obviously we could go lower provided all necessary features are available 

The reason I went with net 5 was from memory it added a number of significant changes. I am happy to go with your suggestion on what TFM is added & if dependency is decreased.

Reply:
Currently, we use STJ as Package in version 7.0.2.
Am I right to assume, that the version of STJ matches the .NET version? (stj7 = net7, stj8 = net8 etc.)

So the trade-off is between how breaking it would be, where dotnet7 would not be breaking at all -
and how many older dotnet version could still profit from this change (everything below the dotnet-version we choose falls back to dotnet-standard).

Is my understanding correct?

In that case, I'd like to go with dotnet 7. To be 100% on the safe side.

Reply:
yes your understanding is correct. So have switched net 5 for net 7 to be 100% safe